### DISCUSSION TITLE
function missing

### DISCUSSION THREAD

**Discussion by likeduck** [PMEAL/porespy#766] [Q&A]
There are two functions in ' porespy.tools ' that are not referenced ( ' label _ boundary _ cells ' and ' pad _ faces ' ). How to solve it ?

**Comment by likeduck** [comment]
when I use 'SNOW_extraction' there is one question in `poresby.tools`
![捕获 PNG2](https://user-images.githubusercontent.com/117707506/204069418-36ba15e8-f880-448c-b893-4167ff86fdda.PNG)
 the code ` label _ boundary _ cells` have changed to `label_boundaries` in `porespy.networks`. Does the code change? how can i get it?

**Reply by jgostick** [reply]
For some reasons, github notifications are appearing as 'read' in my Gmail so I don't notice them.  Sorry for the delay in responding.  Anyway...

I guess you are using an old script with a new version of porespy?  ``pad_faces`` is now called ``add_boundary_regions``.  Perhaps [this example](https://porespy.org/examples/networks/tutorials/adding_boundary_pores.html) will help?

**Reply by likeduck** [reply]
Thanks！

**Reply by jgostick** [reply]
Hey @likeduck, I just added [this example](https://porespy.org/examples/networks/reference/snow2.html), as well as a few others in teh `networks` section.  Hope this is what you needed to get going.

**Reply by likeduck** [reply]
Sorry to reply not in time, thank you for your work.
